srcpkgs/osmin/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'osmin'
+pkgname=osmin
+version=1.10.4
+revision=1
+_libosmscout_version=1.10.4
```

Comment:
judging from https://github.com/janbar/osmin/tree/1.10.4/backend , it uses a commit present in the libosmscout upstream repo (https://github.com/Framstag/libosmscout) and only applies non-functional patches to debug levels...
did you try to package libosmscout?